docs/how-tos/review-a-request.md

```diff
@@ -0,0 +1,176 @@
+
+![More dropdown](../screenshots/more_dropdown_el_request_file.png)
+
+### View context, controls and comments
+The context, controls and comments related to this file's file group can be
+viewed from the file page by clicking on the Context button.
+
+![Context, controls and comments modal](../screenshots/context_modal.png)
+
+### Vote on a file
+
+Use the buttons at the top of the file content to submit your vote
+for this file. Options are:
+
+* **Approve** — output meets disclosure requirements and is safe to be released
+* **Request Changes** — output is not currently acceptable for release. 
+
+After approving or requesting changes to a file, the
+page will display your vote, as well as the overall decision for the
+file. You can change or reset your vote in the same way.
+
+![Request file post-approval](../screenshots/file_approved.png)
+
+You will see the colours and icons change in the file tree to indicate the
+files that you have voted on.
+
+![Request file post-approval](../screenshots/request_tree_post_voting.png)
+
+
+### Add comments
+
+Comments can be added to each file group, to ask questions, or
+provide information on why changes have been requested to any files.
+
+To add a comment, navigate to the file group by clicking on it in
+the file browser. Enter your comment text and click Save.
+
+![Comment form](../screenshots/reviewed_request_comment_in_progress.png)
+
+You can choose to make a comment public and visible to everyone, or 
+private, visible only to other output checkers. In either case, comments are
+hidden from the researcher until the request is returned (or approved/released). 
+
+![Comments on submitted request](../screenshots/reviewed_request_comments.png)
+
+Comments that are created as private can be updated to public at a later stage.
+This can be useful during the consolidation stage, if output checkers agree that
+the comment contains a question to the researcher that they would like to ask
+without revision. 
+
+### Submit your independent review
+
+After you have reviewed all files, you must submit your review. Navigate to
+the request overview page, and click the Submit review button, which will now
+be enabled.
+
+![Submit independent review](../screenshots/submit_review.png)
+
+After your review has been submitted, the request status will change.
+
+![After independent review submitted](../screenshots/submitted_review.png)
+
+
+### Consolidation
+
+Once two independent reviews have been submitted, the request moves into
+"Reviewed" status. At this stage, output checkers are able to see the
+combined decision on files, and comments that they have each made, and
+can discuss and decide how to proceed with the request.
+
+At this stage, more comments can be added or the visibility of existing
+comments can be changed. Output checkers should determine the set of
+comments that they intend to return to the researcher (if any).
```

Comment:
Should we have a section here saying the the reviews need to discuss and agree on the decisions for any files where they disagreed during independent review?

Reply:
Same here - this could link to an explanation doc about independent review. Although that does perhaps stray a bit too far into output checker process, and we don't really want to maintain output checking docs within airlock. Should this link back to the main docs? (I'd been trying not to do that as much as possible, because the main docs aren't accessible from L4 airlock docs)